application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildBanDeleteSpamThreadListener.java

```diff
@@ -0,0 +1,49 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.guild.GuildBanEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jooq.Result;
+
+import org.togetherjava.tjbot.commands.EventReceiver;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.HelpThreadsRecord;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import static org.togetherjava.tjbot.db.generated.Tables.HELP_THREADS;
+
+/**
+ * Implements the functionality to delete recent help threads when the user gets banned.
+ */
```

Comment:
the "implements" part reads a bit weird. also, it doesnt really add anything on top of the class name. so u missed the opportunity to write good and helpful javadoc. maybe:

> Reacts on users being banned from a guild. It then deletes (not just closes) all their recently created help threads.
> 
> This for example tackles threads created by scammers, which are otherwise not automatically cleaned up on ban.

Reply:
done